src/otx/algo/classification/classifier/base_classifier.py

```diff
@@ -220,10 +220,10 @@ def _forward_explain(self, images: torch.Tensor) -> dict[str, torch.Tensor | lis
 
         return {
             "logits": logits,
-            "preds": preds,
-            "scores": scores,
-            "saliency_map": saliency_map,
             "feature_vector": feature_vector,
+            "saliency_map": saliency_map,
```

Comment:
This change seems meaningless.

Reply:
The model API is affected by the order of the corresponding dict output keys.
-> This resolve https://jira.devtools.intel.com/browse/CVS-138604